packages/addons-v5/package.json

```diff
@@ -31,7 +31,8 @@
     "nyc": "^15.1.0",
     "oclif": "3.8.1",
     "proxyquire": "^2.1.0",
-    "sinon": "^6.3.5"
+    "sinon": "^6.3.5",
+    "theredoc": "^1.0.0"
```

Comment:
Just looked this up. I think it could save us hassle if used more widely. Thanks.

Reply:
It was my best attempt to have heredocs in Javascript. The broken indentation on tests was driving me crazy.